perfscale_regerssion_ci/scripts/large_network_policy.sh

```diff
@@ -0,0 +1,64 @@
+#/!/bin/bash
```

Comment:
Possible to add in folders under scripts to be able to separate out the type of functionality/test it is doing

Can set the below as placeholders? 
* network
* performance
* scalability

Reply:
Done